src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACTerminalUnitVariableRefrigerantFlow.cpp

```diff
@@ -218,8 +201,7 @@ namespace energyplus {
     idfObject.setString(ZoneHVAC_TerminalUnit_VariableRefrigerantFlowFields::TerminalUnitAirOutletNodeName, outletNodeName);
 
     auto translateMixer = [&]() {
-      auto t_airLoopHVAC = modelObject.airLoopHVAC();
-      if (t_airLoopHVAC) {
+      if (modelObject.airLoopHVAC() || modelObject.airLoopHVACOutdoorAirSystem()) {
```

Comment:
Don't translate the OutdoorAirMixer if it's on airloopHVAC or airLoopHVACOutdoorAirSystem